extensions/projection/json-schema/schema.json

```diff
@@ -0,0 +1,89 @@
+{
+  "$schema":"http://json-schema.org/draft-07/schema#",
+  "$id":"schema.json#",
+  "title":"Projection Extension",
+  "description":"STAC Projection Extension to a STAC Item",
+  "allOf":[
+    {
+      "$ref": "https://raw.githubusercontent.com/radiantearth/stac-spec/master/item-spec/json-schema/item.json#/definitions/core"
+    },
+    {
+      "$ref":"#/definitions/proj"
+    }
+  ],
+  "definitions":{
+    "proj":{
+      "type": "object",
+      "required": [
+        "properties"
+      ],
+      "properties":{
+        "properties":{
+          "type":"object",
+          "required":[
+            "proj:epsg"
+          ],
+          "properties":{
+            "proj:epsg":{
+              "title":"EPSG code",
+              "type":[
+                "integer",
+                "null"
+              ]
+            },
+            "proj:proj4":{
+              "title":"Coordinate Reference System in PROJ4 format",
+              "type":[
+                "string",
+                "null"
+              ]
+            },
+            "proj:wkt2":{
+              "title":"Coordinate Reference System in WKT2 format",
+              "type":[
+                "string",
+                "null"
+              ]
+            },
+            "proj:projjson":{
+              "title":"Coordinate Reference System in PROJJSON format",
```

Comment:
You could actually link to the official PROJJSON schema here, right?

Reply:
added

Reply:
Sorry, my bad. I forgot the "null" case. It should be: 

```json
"proj:projjson": {
  "oneOf": [
    {
      "$ref": "https://proj.org/schemas/v0.1/projjson.schema.json"
    },
    {
      "type": "null"
    }
  ]
}
```